mmdet3d/models/dense_heads/centerpoint_head.py

```diff
@@ -529,7 +531,7 @@ def get_targets_single(self,
 
                 if width > 0 and length > 0:
                     radius = gaussian_radius(
-                        (length, width),
+                        (width, length),
```

Comment:
Does this modification cause any BC-Breaking, or affect other models' performance?

Reply:
Yeah, I have verified this.